parsers/browscap-js-2/parse

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+var initStart = process.hrtime();
+var Browscap = require('browscap-js');
+var browscap = new Browscap(__dirname + '/node_modules/browscap-json-cache-files/sources/');
```

Comment:
Should be able to remove the directory definition here altogether and let your JS lib find the files based on `require` resolution, which should work for all versions of NPM (doesn't matter where the dependency is installed).  See this commit for an example: https://github.com/diablomedia/useragent-parser-comparison/commit/2c912fbe691676b7c33f945dcc16d6dbb27aa90e